cmake/compiler/arcmwdt/compiler_flags.cmake

```diff
@@ -187,3 +186,6 @@ set_compiler_property(PROPERTY no_position_independent
 
 # Required ASM flags when using mwdt
 set_property(TARGET asm PROPERTY required "-Hasmcpp")
+
+# Pass includes directly into assembly build
+set_property(TARGET asm PROPERTY required "-I${TOOLCHAIN_HOME}/arc/inc")
```

Comment:
Note that here we rewrite previous `required` property value and not appending it.